cmd/airhornbot/bot.go

```diff
@@ -169,12 +169,27 @@ var COW *SoundCollection = &SoundCollection{
 	},
 }
 
+var TRUMP *SoundCollection = &SoundCollection{
+	Prefix: "trump",
+	Commands: []string{
+		"!trump",
+	},
+	Sounds: []*Sound{
+		createSound("mess", 10, 250),
+		createSound("bing", 3, 250),
+		createSound("getitout", 1, 250),
+		createSound("tractor", 3, 250),
+		createSound("worstpres", 3, 250),
```

Comment:
Honestly, I looked but couldn't find a good clip. Find me one and i'll add it :)